packages/gatsby/src/utils/webpack.config.js

```diff
@@ -568,6 +564,11 @@ module.exports = async (
           enforce: true,
         },
       },
+      // We load our pages async through async-requires, maxInitialRequests doesn't have an effect on chunks derived from page components.
+      // By default webpack has set maxAsyncRequests to 6, in some cases this isn't enough an actually makes the bundle size blow up.
+      // We've set maxAsyncRequests to Infinity to negate this. This could potentionally exceed the 25 initial requests that we set before
+      // sadly I do not have a better solution.
+      maxAsyncRequests: Infinity,
```

Comment:
Why not set this to something like 20? Keep the number of potential bundles high but still constrain it? Or is it the extra cpu to recalculate bundles that's causing the slowdown?

Reply:
Problem with constraining it means that we also constrain `import('lodash')` imports, they shouldn't be counted to be part of the initial 25. Sadly, webpack doesn't give me a good way to distinguish `gatsby-component-modules` and dynamic imports.

We might want to look more in-depth into this one. For now, this unblocks slowdown and bad bundles.

Reply:
Cool sounds good 👍 we'll see what happens then